### PR TITLE
feat(icons): add StudyBook and Radar icons

### PR DESCRIPTION
**Pull request by slin12**
### Overview

Adds StudyBookIcon & RadarIcon

Screen Shot 2021-07-14 at 4 09 17 PM
Screen Shot 2021-07-14 at 4 08 27 PM

### PR Checklist

- [ ] Related to designs: https://www.figma.com/file/clAAoUoUroqCNiTg24WRZN/Spaced-Repetition-%2F-Cumulative-Practice?node-id=563%3A1116
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.7.0-alpha.bd5788.0<br/>@codecademy/gamut-kit@0.4.63-alpha.bd5788.0<br/>@codecademy/gamut-labs@19.16.1-alpha.bd5788.0<br/>@codecademy/gamut@34.14.2-alpha.bd5788.0<br/>@codecademy/styleguide@41.14.1-alpha.bd5788.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ef458cd6bb441234b277dd--gamut-preview.netlify.app>https://60ef458cd6bb441234b277dd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ef458cd6bb441234b277dd>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/regular/radar-icon.svg [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/radar-icon.svg:3]: i've had these ids cause some weird side effects lately with other icons. do they still function//look the same when you remove them?

**Review by dreamwasp — APPROVED** [review]
looks good, just small id question//request but once that's sorted 🚀

File: packages/gamut-icons/src/svg/regular/radar-icon.svg (hunk 1 above)
Comment (codecaaron) [reply packages/gamut-icons/src/svg/regular/radar-icon.svg:3]: We can check if this has any affect after SVGR / SVGO processes them - its possible it gets stripped out.  But may want to validate locally.

**Review by codecaaron — APPROVED** [review]

File: packages/gamut-icons/src/svg/regular/radar-icon.svg (hunk 1 above)
Comment (slin12) [reply packages/gamut-icons/src/svg/regular/radar-icon.svg:3]: looks like it gets stripped out - here it is rendered in storybook:

Screen Shot 2021-07-15 at 1 27 43 PM

Comment (slin12) [reply packages/gamut-icons/src/svg/regular/radar-icon.svg:3]: but i can remove the id to be safe 👍🏻

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.7.0-alpha.2852c4.0<br/>@codecademy/gamut-kit@0.4.63-alpha.2852c4.0<br/>@codecademy/gamut-labs@19.16.1-alpha.2852c4.0<br/>@codecademy/gamut@34.14.2-alpha.2852c4.0<br/>@codecademy/styleguide@41.14.1-alpha.2852c4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60f0727ae2500d2e53c4cb60--gamut-preview.netlify.app>https://60f0727ae2500d2e53c4cb60--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60f0727ae2500d2e53c4cb60>Deploy Logs</a></p>

**Pull request merged**